components/monitoring/base/install_prometheus.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: install-prometheus-operator
+objects:
```

Comment:
let's not use openshift template here

Reply:
yes, we don't need Openshift templates here. If anything needs to be changed, it should be handled by Kustomize.